app/models/manageiq/providers/azure/inventory/parser/cloud_manager.rb

```diff
@@ -16,6 +16,7 @@ def parse
     instances
     managed_images
     cloud_databases
+    cloud_database_servers
```

Comment:
Both collections are cached so it won't matter.  In the collector `cloud_databases` calls `*_servers` first already since you have to list databases on a server in the azure api.

Reply:
@agrare @Fryguy I was not able to replicate any other states aside from `"Ready"`. The Azure docs also do not provide the possible states https://docs.microsoft.com/en-us/rest/api/sql/2021-11-01-preview/servers/get#server. As a result, I have left it as https://github.com/ManageIQ/manageiq-providers-azure/pull/513/files#diff-0d177831b5af9863211ff4fa5e8c6810dac1b79b9cf5e132680b8db96f7b6f5cR573-R580. Since deleted servers don't persist on Azure and "Ready" seems to be the only available state, I am wondering whether the `status` column is even necessary at this point. Let me know